### ISSUE TITLE
-bash: /usr/bin/naabu: No such file or directory

### ISSUE THREAD

**Issue by NoneNotNull** [projectdiscovery/naabu#42]
linux amd64, I have moved it to /usr/bin/naabu, but error
-bash: /usr/bin/naabu: No such file or directory

**Comment by NoneNotNull** [comment]
I have resolved it
apt-get install musl-dev

Why not compile it into bin

**Comment by Mzack9999** [comment]
Hi @NoneNotNull - Thanks for reaching out, I think it's been a temporary issue with your environment. As the problem is resolved I'm closing the issue, let us know if you encounter any other problem

**Comment by bauthard** [comment]
This is valid issue across all the FOSS projects.

**Comment by bauthard** [comment]
Related issues @Mzack9999 

```
https://github.com/projectdiscovery/naabu/issues/40
https://github.com/projectdiscovery/subfinder/issues/251
https://github.com/projectdiscovery/naabu/issues/42
https://github.com/projectdiscovery/nuclei/issues/116
```

**Comment by Mzack9999** [comment]
Solved in #51
